### PR TITLE
Suggesting a change to a comment that puzzled me

### PR DESCRIPTION
**Pull request by pcn**
While reading this, the comment made it difficult for me to simply absorb the concept.  It interrupted my reading flow, and I think this expresses the same meaning, but reads a bit better.  It's trivial, but makes it easier for me to move on to the next line.

### CODE REVIEW

File: src/doc/book/references-and-borrowing.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ fn main() {
         return v.iter().fold(0, |a, &b| a + b);
     }
     // Borrow two vectors and sum them.
-    // This kind of borrowing does not allow mutation to the borrowed.
+    // This kind of borrowing does not allow mutation to the borrowed reference.
```
Comment (bluss) [inline src/doc/book/references-and-borrowing.md]: "the borrowed" refers to the vector (or vectors), not to the reference. What could be said is "does not allow mutation *through* the borrowed reference".

Comment (bluss) [inline src/doc/book/references-and-borrowing.md]: Or just "to the borrowed vectors." (?)

Comment (pcn) [inline src/doc/book/references-and-borrowing.md]: I like "through" I think it helps me understand better.

**Comment by steveklabnik** [comment]
@bors: r+ rollup

thanks!

**Pull request merged**